### PR TITLE
Voyage AI embeddings

### PR DESCRIPTION
**Pull request by fzowl**

### CODE REVIEW

File: examples/embed/example_voyageai.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import os
+
+from unstructured.documents.elements import Text
+from unstructured.embed.voyageai import VoyageAIEmbeddingConfig, VoyageAIEmbeddingEncoder
+
+# To use Voyage AI tou will need to pass Voyage AI API Key (obtained from https://dash.voyageai.com/)
```
Comment (Liuhong99) [inline examples/embed/example_voyageai.py]: To use Voyage AI **you** will need to pass Voyage AI API Key (obtained from https://dash.voyageai.com/)

File: docs/source/core/embedding.rst [hunk 2]
```diff
@@ -226,4 +226,55 @@ workload identity, etc…)
...
+embedding vector obtained via this class.
+
+``is_unit_vector`` is a metadata property that denotes if embedding vectors obtained via
+this class are unit vectors.
+
+The following code block shows an example of how to use ``VoyageAIEmbeddingEncoder``. You will
+see the updated elements list (with the ``embeddings`` attribute included for each element),
+the embedding vector for the query string, and some metadata properties about the embedding model.
+
+To use Voyage AI tou will need to pass Voyage AI API Key (obtained from https://dash.voyageai.com/)
```
Comment (Liuhong99) [inline docs/source/core/embedding.rst]: **you**

**Review by Liuhong99 — CHANGES_REQUESTED** [review]
I don't have further questions except for errors from the endpoints.

File: examples/embed/example_voyageai.py (hunk 1 above)
Comment (fzowl) [reply examples/embed/example_voyageai.py]: Hehe, the funny thing is that it is not a misspell but a copy-paste problem. :-)

**Pull request merged**